screenshot-testing-plugin/src/main/kotlin/com/facebook/testing/screenshot/build/PullScreenshotsTask.kt

```diff
@@ -103,6 +103,7 @@ open class PullScreenshotsTask @Inject constructor(
                 .apply {
                     if (verify) {
                         add("--verify")
+                        add("--tolerance=1.5")
```

Comment:
I would like to make it configurable by project that includes this library - I'd be happy with some assistance. I know we can add tolerance as input and concat it to the string here (and wrap it in "if" when it's null), but I could really use some help.